src/main/java/Duke.java

```diff
@@ -1,10 +1,201 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    public Task[] tasks;
+
+    private static final ArrayList<Task> collection = new ArrayList<>();
+
+    public static final Scanner sc = new Scanner(System.in);
+
+
+    public static boolean isNotValid(String str) {
+        String[] s = str.split(" ");
+        return (s.length < 1);
+    }
+
+
+
+
+    public static void main(String[] args) throws DukeException, IOException, IllegalCommandException {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("___________________________________");
+        System.out.println("Hello! I'm Duke\n What can I do for you?");
+        System.out.println("____________________________________");
+
+        String input = sc.nextLine();
+
+        File file = new File("database/duke.txt");
+        if (file.createNewFile()) {
+            System.out.println("File has been created: " + file.getName());
+        } else {
+            System.out.println("File already exists.");
+            Scanner reader = new Scanner(file);
+            while (reader.hasNext()) {
+                reader.useDelimiter(" \\| ");
+                Constants constant = Constants.valueOf(reader.next().toUpperCase());
+                Task task;
+                switch (constant) {
+                    case TODO:
```

Comment:
There should not be indentation for the case clauses. It should look something like this
`switch (constant) {`
 `case TODO:`